rust/kcl-lib/src/std/sketch.rs

```diff
@@ -32,6 +32,9 @@ use crate::{
     },
 };
 
+use super::utils::untype_array;
+use super::utils::untype_point;
```

Comment:
You need to use the nightly rust toolchain for formatting now. We only recently started requiring this in CI.

Reply:
I did this in 7b490da1b45ef57dc3665169b0c19a89aa531c35.